core/storage/app_feedback_report/gae_models_test.py

```diff
@@ -128,7 +134,7 @@ def test_create_and_get_android_report_model(self):
                 None))
 
         report_model = app_feedback_report_models.AppFeedbackReportModel.get(
-            report_id)
+            report_id) # type: app_feedback_report_models.AppFeedbackReportModel # type: ignore[assignment]
```

Comment:
Why do we need to explicitly define type here? (Want to get an idea if this can be seen at other places too)

Reply:
Moved this discussion to the [doc](https://docs.google.com/document/d/1LEWKK2H303eHmWg1hHN2EMIlnewB6EnEyrQoReY-M6s/edit?usp=sharing).